tests/test_app_metadata.py

```diff
@@ -201,6 +201,9 @@ def test_app_metadata(
         # don't check JVM flags in direct comparison, as they might change a bit across machines due to ergonomics
         actual_jvm_flags = metadata["application_metadata"][idx].pop("jvm_flags")
         expected_jvm_flags = expected_metadata.pop("jvm_flags")
+        if is_aarch64():
+            metadata["application_metadata"][idx].pop("libjvm_elfid")
+            expected_metadata.pop("libjvm_elfid")
```

Comment:
Explain in comment why you pop, pls (I suppose because the hash we compare is for x86_64 not for aarch64, so you don't want to compare it?)